src/EpisodeIntegrationService/ReceiveData/ReceiveData.csproj

```diff
@@ -7,6 +7,7 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="CsvHelper" Version="33.0.1" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.19.0" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Http" Version="3.0.13" />
```

Comment:
I don't think we need the Microsoft.Azure.Functions.Worker.Extensions.Http package reference

Reply:
removed